docs/aurora/data-science/frameworks/pytorch.md

```diff
@@ -122,258 +237,176 @@ PyTorch, users generally need to manage casting and loss scaling manually,
 though context managers and function decorators can provide easy tools to do 
 this.
 
-2. PyTorch has a `JIT` module as well as backends to support op fusion, similar 
-to TensorFlow's `tf.function` tools. 
-Please see [TorchScript](https://pytorch.org/docs/stable/jit.html) for more 
-information.
+2. PyTorch has a `JIT` module as well as backends to support op fusion, similar to TensorFlow's `tf.function` tools. See [TorchScript](https://pytorch.org/docs/stable/jit.html) for more information.
```

Comment:
actually above, a typo:

> In general, the way to do this is via the 
PyTorch Automatic Mixed Precision package (AMP), as **descibed** in the